jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalQuartzDBTimerServiceTest.java

```diff
@@ -82,25 +84,22 @@ public GlobalQuartzDBTimerServiceTest(int managerType) {
     }
     
     @Before
-    public void setUp() {
+    public void setUp() throws IOException, SQLException {
         cleanupSingletonSessionId();
-        emf = Persistence.createEntityManagerFactory("org.jbpm.test.persistence");
+        createTimerSchema();
         System.setProperty("org.quartz.properties", "quartz-db.properties");
+        emf = Persistence.createEntityManagerFactory("org.jbpm.test.persistence");
         globalScheduler = new QuartzSchedulerService();
         ((QuartzSchedulerService)globalScheduler).forceShutdown();
     }
     
     @After
-    public void tearDown() {
-        try {
-            
-            globalScheduler.shutdown();
-        } catch (Exception e) {
-            
-        }
-        cleanup();
+    public void tearDown() throws IOException, SQLException {
+        globalScheduler.shutdown();
```

Comment:
Why a possible exception at shutdown is passed through instead of catching it to at least drop tables and cleanup?

Reply:
ok, makes sense